src/EditorFeatures/CSharpTest/ConvertNamespace/ConvertNamespaceRefactoringTests.cs

```diff
@@ -478,6 +477,7 @@ public async Task TextConvertToFileScopedWithCommentedOutContents()
 ",
                 FixedCode = @"
 namespace N;
+
```

Comment:
improvment. the original comment was two lines below the namespace.  we preserve that.